templates/notebook_instance_lifecycle_config/delta_post_compare.go.tpl

```diff
@@ -0,0 +1,2 @@
+delta = modifyDeltaCreate(delta,a,b)
```

Comment:
nit: Can you possibly format this file ex: delta = modifyDeltaCreate(delta, a, b)